locales/bn/json.json

```diff
@@ -573,11 +573,11 @@
     "Sri Lanka": "শ্রীলঙ্কা",
     "Standalone Actions": "স্বতন্ত্র অ্যাকশন",
     "Start Polling": "নির্বাচনের আরম্ভ করুন",
-    "Start polling": "Start polling",
+    "Start polling": "ভোটগ্রহণ শুরু করুন",
     "State / County": "প্রদেশ / দেশ",
     "Stop Impersonating": "ছদ্মবেশী করা বন্ধ করুন",
     "Stop Polling": "ভোটদান বন্ধ করুন",
-    "Stop polling": "Stop polling",
+    "Stop polling": "ভোটগ্রহণ বন্ধ করুন",
```

Comment:
Just a clarification question: is it correct that the translation into Bengali differs from the translation in the `Stop Polling` key one line up?

Reply:
It's all the same

Reply:
Ok, thanks

Reply:
Should I create a new PR?

Reply:
Yes, I already got this one :)